src/shared/api/MatchMinerAPI.ts

```diff
@@ -0,0 +1,86 @@
+import * as request from 'superagent';
+import { ITrial, ITrialMatch } from "shared/model/MatchMiner.ts";
+import { buildCBioPortalAPIUrl } from "./urls";
+
+/**
+ * Retrieves the trial matches for the query given, if they are in the MatchMiner API.
+ */
+
+const cbioportalUrl = buildCBioPortalAPIUrl('api-legacy/proxy/matchminer/api');
+export async function postMatchMinerTrialMatches(query: object): Promise<Array<ITrialMatch>> {
```

Comment:
per comment above, lets rename this following pattern (fetchTrialMatchesUsingPOST)

Reply:
Renamed all apt methods.